src/pca_gophish.yml

```diff
@@ -120,7 +120,34 @@
                 enabled: true
           when:
             - ansible_service_mgr == "systemd"
-    - name: Create GoPhish desktop shortcut for VNC user
+    - name: Create standard directories used by PCA operators
+      block:
+        - name: Create data directories
+          ansible.builtin.file:
+            group: "{{ vnc_username }}"
+            mode: 0755
+            owner: "{{ vnc_username }}"
+            path: /home/{{ vnc_username }}/PCA/Level-{{ item }}/Data
+            state: directory
+            # range is exclusive of end point
+          loop: "{{ range(1, pca_num_levels + 1)|list }}"
+          vars:
+            # The standard number of phishing campaign levels in a PCA
+            pca_num_levels: 6
```

Comment:
Should this be moved down to the main variable definitions at the bottom of this file?

Reply:
Since it's only used in this task, I chose to keep the `pca_num_levels` variable definition nearby.  If it ever gets used elsewhere, I would probably move it down to the main variable definitions.